### PR TITLE
Added example yaml config files

### PR DESCRIPTION
**Pull request by cooleel**
Added 2 pipeline yaml config files and these are fully working with the universal scorer. 

related issues:
https://github.com/h2oai/argus-ocr/issues/636 
https://github.com/h2oai/argus-ocr/issues/713
https://github.com/h2oai/argus-ocr/pull/792

### CODE REVIEW

File: pipeline_config/pipeline-conditional-processing.yaml
Comment (smg478) [inline pipeline_config/pipeline-conditional-processing.yaml:1]: Can we make this version cleaner by removing the commented code and inserting a brief overview of the pipeline? For example, what it does in plain language.

We can store these lines of code in a separate file probably. Your call.

File: pipeline_config/pipeline-conditional-processing.yaml
Comment (smg478) [inline pipeline_config/pipeline-conditional-processing.yaml:30]: It would be helpful to describe what tasks can be done, and what key-values are available for the user. I would suggest removing generic comments from each individual line to make it look cleaner and easy to follow.

File: pipeline_config/pipeline-kyc-id.yaml
Comment (smg478) [inline pipeline_config/pipeline-kyc-id.yaml:81]: I would recommend using 0.2 as the default threshold for the token labeling model. Because after token merging and probability averaging, some entities get pretty low confidences, and they are correct.

File: pipeline_config/pipeline-conditional-processing.yaml
Comment (cooleel) [reply pipeline_config/pipeline-conditional-processing.yaml:1]: These are the command line calls that are equivalent to the yaml file, that's why I put them here as a reference. But you are right, I can remove them out to another place, e.g. notes.

File: pipeline_config/pipeline-conditional-processing.yaml
Comment (cooleel) [reply pipeline_config/pipeline-conditional-processing.yaml:30]: I will add some into the yaml file, but most of the above questions I think should go to argus' documentation?

**Review by smg478 — APPROVED** [review]

**Pull request merged**
